godo.go

```diff
@@ -174,6 +180,11 @@ func (c *Client) NewRequest(method, urlStr string, body interface{}) (*http.Requ
 	return req, nil
 }
 
+// Set DO API request completion callback
+func (c *Client) OnRequestCompleted(rc RequestCompletionCallback) {
```

Comment:
Please document exported items in accordance with http://blog.golang.org/godoc-documenting-go-code.